cookiecutter-django-app/{{cookiecutter.repo_name}}/Makefile

```diff
@@ -85,11 +85,10 @@ selfcheck: ## check that the Makefile is well-formed
 
 extract_translations: ## extract strings to be translated, outputting .mo files
 	rm -rf docs/_build
-	cd {{cookiecutter.app_name}} && ../manage.py makemessages -l en -v1 -d django
-	cd {{cookiecutter.app_name}} && ../manage.py makemessages -l en -v1 -d djangojs
+	cd {{cookiecutter.app_name}} && i18n_tool extract --no-segment
 
 compile_translations: ## compile translation files, outputting .po files for each supported language
-	cd {{cookiecutter.app_name}} && ../manage.py compilemessages
+	cd {{cookiecutter.app_name}} && i18n_tool generate
 
 detect_changed_source_translations:
```

Comment:
`pull_translations` needs to enable `atlas pull` behind if the environment variable `OPENEDX_ATLAS_PULL` is set:

 - See: https://github.com/openedx/credentials/pull/2153

Reply:
@brian-smith-tcril should we enable `atlas pull` exclusively? or we should keep it disabled behind the `OPENEDX_ATLAS_PULL=yes` environment variable?

cc: @shadinaif

Reply:
I'd say we should keep it disabled behind the env var until we're fully ready to migrate everything over. That'll keep things working in the off chance someone creates a new repo using the cookiecutter between now and then.

Reply:
done

Reply:
thanks!!